docs/errors/E202.md

```diff
@@ -0,0 +1,30 @@
+# E220: semicolon after else may be causing unexpected behavior.
+
+A semicolon next to the `else` keyword may cause its body to be executed even when previous `if` statement condition is true.
+
+    if (true) {
+        console.log("true");
+    } else; {
+        console.log("else");
+    }
+
+Example output
+```
+> true
+> else
+```
+
+To avoid this behavior, remove the semicolon between the else keyword and the opening brace of the else block.
```

Comment:
Maybe:
`In case this is impacting your code, you should remove the semicolon next to the else keyword`